### PR TITLE
Fixing the issue interpolateQuery

### PR DESCRIPTION
**Pull request by aa6my**
**Issue**
```
Type: Error
Message: Call to undefined method CI_Profiler::interpolateQuery()
Filename: .../application/third_party/codeigniter-forensics/libraries/Profiler.php
Line Number: 256
```

**Successfully running**
eloquent

**Reminder**
Make sure enable query log in database.php in config to make it successfully running.

```
use Illuminate\Database\Capsule\Manager as Capsule;

$capsule = new Capsule;

$capsule->addConnection(array(
	'driver'    => 'mysql',
	'host'      => $db['default']['hostname'],
	'database'  => $db['default']['database'],
	'username'  => $db['default']['username'],
	'password'  => $db['default']['password'],
	'charset'   => 'utf8',
	'collation' => 'utf8_unicode_ci',
	'prefix'    => $db['default']['dbprefix'],
));

// Make this Capsule instance available globally via static methods... (optional)
$capsule->setAsGlobal();

// Setup the Eloquent ORM... (optional, unless you've used setEventDispatcher())
$capsule->bootEloquent();

// Enable Query Log, if not enable Profiler wont show query.
$connection = $capsule->getConnection();
$connection->enableQueryLog();
```

### CODE REVIEW

File: config/profiler.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 $config['memory_usage']         = TRUE;
 $config['post']                 = TRUE;
 $config['queries']              = TRUE;
-$config['eloquent']             = FALSE;
+$config['eloquent']             = TRUE;
```
Comment (lonnieezell) [inline config/profiler.php]: We don't want Eloquent enabled by default. Looks fine otherwise.

Comment (aa6my) [inline config/profiler.php]: but those effect when common people using composer. the config active as git repo in thirdparty. so what the best solution to trigger TRUE and FALSE for enable eloquent ?

Comment (aa6my) [inline config/profiler.php]: @lonnieezell can close and merge.

**Comment by aa6my** [comment]
Done make set default to false

**Comment by lonnieezell** [comment]
Great. Thanks!

**Pull request merged**